### ISSUE TITLE
Error when selecting level

### ISSUE THREAD

**Issue by eggnima** [workshopper/stream-adventure#211]
Hi, when selecting a level via the menu or the `stream-adventure select xxx` command, I'm getting the following error:

> fs.js:124
>   throw new ERR_INVALID_CALLBACK();
>   ^
> 
> TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
>     at maybeCallback (fs.js:124:9)
>     at Object.fs.writeFile (fs.js:1244:14)
>     at Shop.save (/usr/local/lib/node_modules/stream-adventure/node_modules/adventure/index.js:288:8)
>     at Shop.select (/usr/local/lib/node_modules/stream-adventure/node_modules/adventure/index.js:246:10)
>     at Shop.execute (/usr/local/lib/node_modules/stream-adventure/node_modules/adventure/index.js:97:14)
>     at Object.<anonymous> (/usr/local/lib/node_modules/stream-adventure/bin/cmd.js:18:6)
>     at Module._compile (internal/modules/cjs/loader.js:678:30)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)
>     at Module.load (internal/modules/cjs/loader.js:589:32)
>     at tryModuleLoad (internal/modules/cjs/loader.js:528:12)

I'm running v10.0.0 of node, on macOS v10.13.4 (17E199), with iTerm2 v3.1.6.

**Comment by i474232898** [comment]
Use v8.11.1 instead

**Comment by Drivez** [comment]
@eggnima you can use nvm to have multiple node versions on you machine 
TLDR;
Install nvm on mac, run 
`curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash`

then open a new tab(terminal window) and install a node version 
`nvm install 8.11.1`
and add the 10 version 
`nvm install 10`

now you can switch by using `nvm use 8`
and switch back `nvm use 10`

[Link to nvm repo ](https://github.com/creationix/nvm)
